draft-gazdag-x509-shbs.md

```diff
@@ -602,62 +479,31 @@ This ASN.1 Module builds upon the conventions established in [RFC5911].
-       KEY SPHINCS-Plus-PublicKey
-       PARAMS ARE absent
-       CERT-KEY-USAGE
-          { digitalSignature, nonRepudiation, keyCertSign, cRLSign } }
-
-    -- pk-sphincs-plus-192 is defined in [I-D.ietf-lamps-cms-sphincs-plus]
-
-    pk-sphincs-plus-192 PUBLIC-KEY ::= {
-       IDENTIFIER id-alg-sphincs-plus-192
-       KEY SPHINCS-Plus-PublicKey
-       PARAMS ARE absent
-       CERT-KEY-USAGE
-          { digitalSignature, nonRepudiation, keyCertSign, cRLSign } }
-
-    -- pk-sphincs-plus-256 is defined in [I-D.ietf-lamps-cms-sphincs-plus]
-
-    pk-sphincs-plus-256 PUBLIC-KEY ::= {
-       IDENTIFIER id-alg-sphincs-plus-256
-       KEY SPHINCS-Plus-PublicKey
-       PARAMS ARE absent
-       CERT-KEY-USAGE
-          { digitalSignature, nonRepudiation, keyCertSign, cRLSign } }
-
-    SPHINCS-Plus-PublicKey ::= OCTET STRING
-
     END
 
     <CODE ENDS>
 
 # Security Considerations
 
-The security requirements of [NIST-SP-800-208] and [NIST-FIPS-205] MUST be
-taken into account.
+The security requirements of [SP800208] MUST be taken into account.
 
-For the stateful HBS (HSS, XMSS, XMSS^MT) it is crucial to stress the
-importance of a correct state management. If an attacker were able to obtain
-signatures for two different messages created using the same OTS key, then it
-would become computationally feasible for that attacker to create forgeries
-[BH16]. As noted in [MCGREW] and [ETSI-TR-103-692], extreme care needs to be
-taken in order to avoid the risk that an OTS key will be reused accidentally.
-This is a new requirement that most developers will not be familiar with and
-requires careful handling.
+For S-HBS it is crucial to stress the importance of a correct state management.
+If an attacker were able to obtain signatures for two different messages
+created using the same OTS key, then it would become computationally feasible
+for that attacker to create forgeries [BH16]. As noted in [MCGREW] and
+[ETSI-TR-103-692], extreme care needs to be taken in order to avoid the risk
+that an OTS key will be reused accidentally.  This is a new requirement that
+most developers will not be familiar with and requires careful handling.
 
 Various strategies for a correct state management can be applied:
 
 - Implement a track record of all signatures generated by a key pair associated
-  to a stateful HBS instance. This track record may be stored outside the
+  to a S-HBS instance. This track record may be stored outside the
   device which is used to generate the signature. Check the track record to
   prevent OTS key reuse before a new signature is released. Drop the new
   signature and hit your PANIC button if you spot OTS key reuse.
 
-- Use a stateful HBS instance only for a moderate number of signatures such
+- Use a S-HBS instance only for a moderate number of signatures such
   that it is always practical to keep a consistent track record and be able to
   unambiguously trace back all generated signatures.
```

Comment:
Here is a good point where we can reference the new draft on state and backup management for S-HBS